File: Multiplicar.py
# coding=utf8
def multiplicar(num):
    for x in range(1,11):
        print(num*x);

n1 = int(input("Ingrese el Primer Numero : "))
n2 = int(input("Ingrese el Segundo Numero: "))

if(n1>n2):
    print("El Primer Numero Ingresado es Mayor");
    multiplicar(n1);  
elif(n2>n1):
    print("El Segundo Numero Ingresado es Mayor");
    multiplicar(n2);  
else:
    print("Los dos Numeros Ingresados son Iguales");
    multiplicar(n1);  

File: Multiplicar.cpp
#include <conio.h>
#include <stdio.h>

void multiplicar(int num){
	for (int i = 1 ; i <= 10 ; i++ )
    	printf( "\n   %d * %d = %d", i, num, i * num );
}

int main()
{
    int n1, n2;

    printf( "\n   Ingrese el Primer Numero: ");
    scanf( "%d", &n1 );
    printf( "\n   Ingrese el Segundo Numero: ");
    scanf( "%d", &n2 );

    if ( n1 > n2 ){
        printf( "\n   El Primer Numero Ingresado es mayor.", n1 );
        multiplicar(n1);
    }
        
    else
        if ( n1 < n2 ){
			printf( "\n   El Segundo Numero Ingresado es mayor.", n2 );
        	multiplicar(n2);
       }
        else{
            printf( "\n   Los Dos Numeros Ingresados Son Iguales" );
        	multiplicar(n1);
		}    
 
    getch(); /* Pausa */

    return 0;
}
